molgenis-navigator/src/main/resources/l10n/navigator_en.properties

```
@@ -9,4 +9,6 @@ delete-confirmation-text=Are you sure you want to permanently remove these items
 delete-confirmation-title=Confirm Deletion
 delete-confirmation-ok-text=Delete
 delete-confirmation-cancel-text=Cancel
-
+progress-download-running=Started preparing download...
+progress-download-success=Finished preparing download.
+NAV01=Download failed: {0}
```

```suggestion
N03=Download failed: {0}
```
To harmonize with Copy Exceptions